content/en/security/suppressions.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Suppressions
+kind: documentation
+disable_toc: false
+further_reading:
+- link: "security/detection_rules/"
+  tag: "Documentation"
+  text: "Learn more about detection rules"
+---
+
+<div class="alert alert-warning">Suppressions are available for Cloud Workload Security and Cloud SIEM.</div>
+
+## Overview
+
+Suppressions are specific conditions for when a signal should not be generated, which can improve the accuracy and relevance of the signals that are generated.
+
+## Suppression routes
+
+Suppressions can be set up for individual detection rules and also more broadly across multiple detection rules with suppression rules.
+
+### Detection rules
+
+When you [create][1] or [modify][2] a detection rule, you can define a suppression query to prevent a signal from getting generated. For example, add a rule query to determine when a detection rule triggers a security signal. You can also customize the suppression query to suppress signals for a specific attribute value.
+
+{{< img src="security/security_monitoring/suppressions/detection_suppression_rule.png" alt="The detection rule editor showing the add suppression query section" style="width:65%;" >}}
+
+### Suppression rules
+
+Use suppression rules to set general suppression conditions across multiple detection rules and signals instead of setting up suppression conditions for each individual detection rules. For example, you can set up a suppression rule to suppress any signal that contains a specific IP.
```

Comment:
```suggestion
Use suppression rules to set general suppression conditions across multiple detection rules instead of setting up suppression conditions for each individual detection rules. For example, you can set up a suppression rule to suppress any signal that contains a specific IP.
```